### PR TITLE
Documenting config/limitations/contributing

### PR DESCRIPTION
**Pull request by Ocramius**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+You can set up symmetric key based signatures via the
+`StorageLessSession::fromSymmetricKeyDefaults` named constructor:
+
+```php
+use StoragelessSession\Http\StorageLessSession;
+
+$sessionMiddleware = StorageLessSession::fromSymmetricKeyDefaults(
+    'contents of the symmetric key', // symmetric key
+    1200                             // session lifetime, in seconds
```
Comment (lcobucci) [inline docs/configuration.md]: Should we add the refresh time parameter to the named constructors?

**Comment by lcobucci** [comment]
@Ocramius :+1:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+To install the project and run the tests, you need to clone it first:
+
+```sh
+$ git clone git://github.com/Ocramius/PSR7Session.git
+```
+
+You will then need to run a composer installation:
+
+```sh
+$ cd ProxyManager
```
Comment (danizord) [inline CONTRIBUTING.md]:
```sh
$ cd PSR7Session
```

Comment (danizord) [inline CONTRIBUTING.md]: Fixed by https://github.com/Ocramius/PSR7Session/pull/18

**Pull request merged**